apps/engmanagement/src/pages/[slug].tsx

```diff
@@ -195,9 +207,7 @@ export const getServerSideProps: GetServerSideProps = async (context: any) => {
     slug: currentPost.slug,
   })
 
-  const authorized = initialData.subscribersOnly
-    ? await checkSubscriber(context, initialData.ckTagId)
-    : true
+  const authorized = initialData.subscribersOnly ? !isEmpty(convertkitId) : true
```

Comment:
I think if we have a CK id **at all** we can just show it and no need to check async